### PR TITLE
UWP Validation Tests

### PR DESCRIPTION
**Pull request by CedricGuillemet**
UWP Validation Test App. To run locally until it's automated with the CI.

### CODE REVIEW

File: Apps/ValidationTests/Shared/TestUtils.h [hunk 1]
```diff
@@ -79,7 +94,10 @@ namespace Babylon
             const int32_t exitCode = info[0].As<Napi::Number>().Int32Value();
             doExit = true;
             errorCode = exitCode;
-#if ANDROID
+#if defined(__cplusplus_winrt)
+            // ceguille: I didn't find a better way to do it for UWP
+            exit(errorCode);
```
[the remark this answers is not in the document]
Comment (CedricGuillemet) [reply Apps/ValidationTests/Shared/TestUtils.h:102]: "This method can be called only from the thread that created the Application object."

[the remark this answers is not in the document]
Comment (bghgary) [reply Apps/ValidationTests/Shared/TestUtils.h:89]: I think you use the CoreDispatcher to get on to the right thread?

[the remark this answers is not in the document]
Comment (CedricGuillemet) [reply Apps/ValidationTests/Shared/TestUtils.h:89]: I've spent 1h trying to do it properly...without success.

[the remark this answers is not in the document]
Comment (bghgary) [reply Apps/ValidationTests/Shared/TestUtils.h:89]: I send the wrong link for the exit function, but it appears there is no way to specify the exit code, so this won't work.

**Review by syntheticmagus — APPROVED** [review]

**Pull request merged**